chapters/03-internals.md

```diff
@@ -352,7 +352,7 @@ var Todo = Backbone.Model.extend({
 
   validate: function(attributes){
     if(attributes.title === undefined){
-        return "Remember to set a title for your todo.";
+        return console.log(”Remember to set a title for your todo.”);
```

Comment:
I see now how the return is handled for the second validation example, so I go ahead and close this PR, though for the first example I struggled for a minute to figure out why nothing was being logged. Thanks for the comments!